UnityProject/Assets/Scripts/Consoles/CommsConsole.cs

```diff
@@ -3,7 +3,7 @@
 using UnityEngine;
 using UnityEngine.Events;
 
-public class CommsConsole : NBHandApplyInteractable
+public class CommsConsole : MonoBehaviour, ICheckedInteractable<HandApply>
```

Comment:
This was a network behavior before, does it still need to be one?

Reply:
I don't think so, it doesn't have any `[Server]` methods or syncvars or anything else NetworkBehaviourish